components/expenses/ExpenseForm.js

```diff
@@ -597,7 +609,7 @@ const ExpenseFormBody = ({
                 </StyledButton>
                 {errors.payoutMethod?.data?.currency && touched.items?.some?.(i => i.amount) && (
                   <Box mx={[2, 0]} mt={2} color="red.500" fontSize="12px" letterSpacing={0}>
-                    {errors.payoutMethod.data.currency.toString()}
+                    {errors.payoutMethod.data.currency}
```

Comment:
According to https://github.com/opencollective/opencollective-frontend/blob/1ef8d90e2ddeefe6d2928176c966236cdc8dc3ce/components/expenses/PayoutMethodForm.js#L50, this can be something else than a string. Are you sure about removing this `.toString()`? What's the reason for this change?

Reply:
I think this is a mistake and hence I've added the `.toString()` back. 🤔